bin/tictactoe

```diff
@@ -1,3 +1,77 @@
 #!/usr/bin/env ruby
 
```

Comment:
Thanks Cernan! I thought that it looked bad in the bin file, but for some reason creating a new class didn't occur to me. I've updated the files!